api_endpoints/v1_postcode_lookup/app.py

```diff
@@ -13,9 +14,24 @@
 
 init_sentry()
 
+if logger_arn := os.environ.get("LOGGER_ARN"):
+    POSTCODE_LOGGER = DCWidePostcodeLoggingClient(function_arn=logger_arn)
+else:
+    POSTCODE_LOGGER = DCWidePostcodeLoggingClient(
+        function_arn="fake", fake=True
```

Comment:
Is `function_arn` kwarg required? (you haven't included it [here](https://github.com/DemocracyClub/WhoCanIVoteFor/pull/1733/commits/f85faf032c31c4481482cff86afe247777533179#diff-02457a2c1d6abee51336f55448fbf8e0655f8d109c0c2ba7006577fd0857c0a7R294):